database_admin/migrations/058_mark_stale_fix.up.sql

```diff
@@ -0,0 +1,73 @@
+CREATE OR REPLACE FUNCTION delete_systems(ids INT[])
```

Comment:
As systems are identified by "id" it requires full scan because we don't have index for id. It would be better to either use "inventory_id" as we have index for that or couple (rh_account_id, id). This method is only called from "delete_culled_systems" method so it wouldn't be hard to modify the method head a bit.